translate/translate.go

```diff
@@ -0,0 +1,40 @@
+package translate
+
+import (
+	"context"
+
+	translate "github.com/yandex-cloud/go-genproto/yandex/cloud/ai/translate/v2"
+	"google.golang.org/grpc"
+)
+
+// TranslateServiceClient -
+type TranslateServiceClient struct {
```

Comment:
Since this name corresponds to the service name, it should be TranslationServiceClient